plugins/outputs/cloudrun/coverage.out

```diff
@@ -0,0 +1,39 @@
+mode: set
+github.com/influxdata/telegraf/plugins/outputs/cloudrun/cloudrun.go:70.70,72.2 1 1
+github.com/influxdata/telegraf/plugins/outputs/cloudrun/cloudrun.go:74.37,75.22 1 1
+github.com/influxdata/telegraf/plugins/outputs/cloudrun/cloudrun.go:79.2,79.12 1 0
+github.com/influxdata/telegraf/plugins/outputs/cloudrun/cloudrun.go:75.22,77.3 1 1
+github.com/influxdata/telegraf/plugins/outputs/cloudrun/cloudrun.go:82.48,88.16 5 1
+github.com/influxdata/telegraf/plugins/outputs/cloudrun/cloudrun.go:92.2,93.16 2 1
```

Comment:
Please remove this file.

Reply:
@zachmares do you know what this is for, I don't recognize it.

Reply:
This is a test coverage file. It can be safely removed.

Reply:
Thanks @powersj